tests/behat/features/capabilities/event/filter-community-events.feature

```diff
@@ -0,0 +1,50 @@
+@api @javascript @event @stability @stability-1 @filter-community-events
+Feature: Filter community events
+  Benefit: Ability to filter events on All events page
+  Role: As a Verified
+  Goal/desire: I want to filter events that have all day checkbox activated
+
+  Scenario: Successfully use filters for ongoing and upcoming events with today's end date
+    Given users:
+      | name               | mail             | status | timezone | roles    |
+      | regular_user       | some@example.com | 1      | UTC      | verified |
+    Given events with non-anonymous author:
+      | title                     | body        | field_content_visibility | field_event_date    | field_event_date_end | field_event_all_day	 | langcode |
+      | Upcoming or Ongoing Event | lorem ipsum | public                   | today               | today                | 1                    | en       |
+    When I am logged in as "regular_user"
```

Comment:
Here you're arranging, so we can signal that with one `Given` and use `and` to indicate the next step is still part of this block

I've also changed from creating the test user and logging in as them to just logging in as a random user with the verified role. The reason is that in this test we care more about the user's permissions than the specific user.

```suggestion
    Given users:
      | name               | mail             | status | timezone | roles    |
      | regular_user       | some@example.com | 1      | UTC      | verified |
    And events with non-anonymous author:
      | title                     | body        | field_content_visibility | field_event_date    | field_event_date_end | field_event_all_day	 | langcode |
      | Upcoming or Ongoing Event | lorem ipsum | public                   | today               | today                | 1                    | en       |
    And I am logged in as "regular_user"
```

The user we're using can go in either our arrange or our act block. Most tests currently include it in the arrange block.